src/components/ChatAiWidget.tsx

```diff
@@ -137,33 +126,18 @@ const ChatAiWidget = ({
     }
   }, []);
 
-  assert(
-    applicationId !== null && botId !== null,
-    'applicationId and botId must be provided'
-  );
-
   return (
-    <ConstantStateProvider
-      // If env is not provided, prop will be used instead.
-      // But Either should be provided.
-      applicationId={CHAT_WIDGET_APP_ID ?? applicationId}
-      botId={CHAT_WIDGET_BOT_ID ?? botId}
-      {...constantProps}
-    >
-      <HashedKeyProvider hashedKey={hashedKey ?? null}>
```

Comment:
These providers have been moved to Chat component for the consistency. And also, I needed to export the `Chat` component to make it possible to be used from outside.